csrc/ir/internal_nodes.h

```diff
@@ -2288,4 +2288,51 @@ class MatmulOp : public Expr {
       const std::vector<PolymorphicValue>& inputs) const override;
 };
 
+// Linear Operator to be expression evaluated without decomposition.
```

Comment:
I may have this wrong, but later we may decide to evaluate this with nvfuser?
If so, we may not need to mention anything here.